apis/Google.Cloud.Vision.V1/Google.Cloud.Vision.V1/AnnotateImageResponsePartial.cs

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Not a real change
```

Comment:
Is this meant to be here?

Reply:
Not by the time I merge, no. But I need it to be in the PR so that I can check that detect-pr-changes.sh gives the right output.

Reply:
ok, thanks